### DISCUSSION TITLE
Puanım var ama coinim yok...

### DISCUSSION THREAD

**Discussion by ghost** [IBAX-io/go-ibax#810] [Q&A]
...nasıl olacak? teşekkürler.

**Comment by Skips1** [comment]
Puanına göre Coin verecekler kanka En çok puanı alan en çok ödülü alır tarzı kanka.

**Reply by ghost** [reply]
Teşekkür ederim. Peki ne zaman?

**Reply by Skips1** [reply]
Kanka 1 şubattan sonra başlıyacak dağıtım ama herşey çok yeni bu kadar aceleci olmamak lazım :)

**Reply by ghost** [reply]
Tamam o zaman arkamıza yaslanıp bekliceğiz. :) Cüzdanları bağlayın. ;)

**Reply by Skips1** [reply]
:) :) :) Lets gooooo

**Comment by hsynkader** [comment]
Puana göre coin dağılımı nasıl olacak belli değil. Coin almanın başka bir yolu da ref yapmak.
